tests/utils/shippable/shippable.sh

```diff
@@ -68,12 +68,18 @@ cd "${TEST_DIR}"
 
 # START: HACK install dependencies
 retry ansible-galaxy -vvv collection install ansible.netcommon
-retry ansible-galaxy -vvv collection install ansible.posix
-retry ansible-galaxy -vvv collection install community.crypto
-retry ansible-galaxy -vvv collection install community.internal_test_tools
 retry ansible-galaxy -vvv collection install community.kubernetes
 retry ansible-galaxy -vvv collection install google.cloud
 
+if [ "${script}" != "sanity" ] || [ "${test}" == "sanity/extra" ]; then
+    retry ansible-galaxy -vvv collection install community.internal_test_tools
```

Comment:
```suggestion
    # Nothing further should be added to this list.
    # This is to prevent modules or plugins in this collection having a runtime dependency on other collections.
    retry ansible-galaxy -vvv collection install community.internal_test_tools
```